### PR TITLE
Mention 2i2c as sponsoring aarch64 self-hosted runners

### PR DESCRIPTION
**Pull request by mathbunnyru**
## Issue ticket if applicable

Fix: https://github.com/jupyter/docker-stacks/issues/2002

## Checklist (especially for first-time contributors)

- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] I will try not to use force-push to make the review process easier for reviewers
- [ ] I have updated the documentation for significant changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ more information is available in the [documentation](https://jupyter-docker-stac
 - Single-platform images have either `aarch64-` or `x86_64-` tag prefixes, for example, `jupyter/base-notebook:aarch64-python-3.10.5`
 - Starting from `2022-09-21`, we create multi-platform images (except `tensorflow-notebook`)
 - Starting from `2023-06-01`, we create multi-platform `tensorflow-notebook` image as well
+- Starting from `2023-10-31`, `aarch64` images are sponsored by amazing [`2i2c project`](https://2i2c.org)
```
Comment (mathbunnyru) [inline README.md:93]:
```suggestion
- Starting from `2023-10-31`, `aarch64` self-hosted runners are sponsored by amazing [`2i2c project`](https://2i2c.org)
```

**Comment by mathbunnyru** [comment]
From this [comment](We'd like to be credited with a simple line in the docs and README as sponsoring the CI for ARM images.).

> We'd like to be credited with a simple line in the docs and README as sponsoring the CI for ARM images.

We have a nice feature, that our README.md is actually the prefix of our docs, so after this PR the line will be present in both docs and README.md
It's a [cool feature](https://github.com/jupyter/docker-stacks/blob/main/docs/index.rst?plain=1#L1), right? 🙂

**Comment by yuvipanda** [comment]
That is pretty cool :)

I would *love* for it to be slightly higher up in the README if you think it's appropriate!

**Comment by mathbunnyru** [comment]
> That is pretty cool :)
> 
> I would _love_ for it to be slightly higher up in the README if you think it's appropriate!

I created a separate section, which makes this line more noticeable (and moves it slightly higher).
I wouldn't like to move it even higher, because I want our users to first see examples and other user-related sections and only then more project-specific information.

Merging this one ❤️

**Comment by yuvipanda** [comment]
This looks awesome, thank you @mathbunnyru!

**Pull request merged**